examples/Recipies/initialization/modules/correctly_initialized_config.py

```diff
@@ -0,0 +1,14 @@
+"""Configuration for initial_value with proper loading of the config file via the cli in the Config module"""
+
+from application_settings import ConfigBase, config_filepath_from_cli, dataclass
+
+
+@dataclass(frozen=True)
+class CorrectlyInitializedConfig(ConfigBase):
```

Comment:
It's now pretty difficult to truly grasp the difference between the two configs, because they are 'so far apart'. For such a small example I think it's way clearer to limit the number of files.

Meaning I think we should have a single `my_module.configs.py` file that defines both configs (and easily show they are equal, could even use empty inheritance for that), and nicely show that you `load` one, and add a comment you're not doing that for the other.